ONSdigital/respondent-home-ui
# T006 Remove static files and disable file routing

# Motivation and Context
Now that RH makes use of the CDN, the static files in the repository and the temporary server can be removed as they are no longer used when rendering the form.

# What has changed
- Disabled static file serving route on app creation
- Deleted static files and directories
- Removed redirect on a 403

# How to test?
Run respondent home locally and check nothing visually has changed.
Bonus points: check the network tab in Chrome dev tools for any 404's.
